Lib/enum.py

```diff
@@ -1551,6 +1559,15 @@ def _power_of_two(value):
         return False
     return value == 2 ** _high_bit(value)
 
+def _dataclass_repr(self):
+    dcf = self.__dataclass_fields__
+    return ', '.join(
+            '%s=%r' % (k, getattr(self, k))
+            for k in dcf.keys()
+            if dcf[k].repr
+            )
+
+
 def global_enum_repr(self):
```

Comment:
```suggestion
            )

def global_enum_repr(self):
```

Tiny nitpick, but looks like there's an extra line break here